packages/storybook/src/stories/index.js

```diff
@@ -1,5 +1,11 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
 import 'normalize.css';
 import './styles.css';
+import readme from '../../../../README.md';
+
+// eslint-disable-next-line react/no-danger
+storiesOf(' Welcome', module).add('to govuk-react', () => <div style={{ padding: '10px' }} className="markdown-body" dangerouslySetInnerHTML={{ __html: readme }} />);
```

Comment:
is this not just setting raw markdown as the HTML content?

in other words, it feels like this won't actually be rendering the markdown content

we can probably use `react-markdown` to ensure that the MD gets rendered as proper HTML
that would remove the need for the `react/no-danger` lint disable

Reply:
Yes - I had a look at using `react-markdown` but the funny thing is when the file is displayed it is _not_ markdown but is already rendered as HTML! However, the HTML is just rendered into the page unless we use `dangerouslySetInnerHTML`.

Fire it up and take a look...